storage/core/src/server/Interfaces.ts

```diff
@@ -17,9 +17,21 @@ export type TransferType = "buffer" | "stream" | "local";
 export type TransferData = Readable | Buffer | string;
 export type MultipartUploadData = Readable | string;
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type AbortSignalListener = (this: AbortSignal, ev: any) => any;
+
+export interface AbortSignal {
```

Comment:
Since this interface also doesn't follow the current specification, it seems like it would require breaking changes in the future, once one of the dependencies updates how they define AbortSignal. It's missing [reason](https://nodejs.org/api/globals.html#abortsignalreason) property and [throwIfAborted()](https://nodejs.org/api/globals.html#abortsignalthrowifaborted) method.